src/kernel/src/process/module/recompiler.rs

```diff
@@ -158,11 +158,14 @@ impl<'input> Recompiler<'input> {
                 Code::Mov_rm8_r8 => (self.transform_mov_rm8_r8(i), false),
                 Code::Mov_rm32_r32 => (self.transform_mov_rm32_r32(i), false),
                 Code::Mov_rm64_r64 => (self.transform_mov_rm64_r64(i), false),
-                Code::Nop_rm16 | Code::Nop_rm32 | Code::Nopw => (self.preserve(i), false),
+                Code::Movzx_r32_rm8 => (self.preserve(i), false),
```

Comment:
We can't preserve this one because the source operand may use RIP-relative, which we do not supported yet.